### PR TITLE
Fix #407: Order history entry target permissions consistently.

### LINKED ISSUE #407: Order permissions before diff?
We should not show a diff for ther principals who weren't changed:

screenshot from 2017-03-15 17-09-48

I'm not sure that the server guarantees the order of principals in the permissions. Maybe it should (*please open issue*).

In the meantime we could order them alphabetically...

### PR DESCRIPTION
**Pull request by n1k0**
Refs #407, ensure sorting history entry permissions list before rendering a diff.

### CODE REVIEW

**Comment by n1k0** [comment]
Hold on, I'm gonna add a few unit tests for that function introduced in `utils`.

File: src/utils.js [hunk 1]
```diff
@@ -239,3 +239,22 @@ export function scrollToBottom(): Promise<void> {
...
   return Promise.resolve();
 }
+
+export function sortHistoryEntryPermissions(
+  entry: ResourceHistoryEntry
+): ResourceHistoryEntry {
+  if (entry.action === "delete") {
+    return entry;
+  }
+  const { permissions } = entry.target;
```
Comment (glasserc) [inline src/utils.js]: Rather than checking explicitly for `delete`, should we just check for the absence of `permissions`?

Comment (n1k0) [inline src/utils.js]: Good idea, will update.

**Comment by n1k0** [comment]
Released in [v1.12.0](https://github.com/Kinto/kinto-admin/releases/tag/v1.12.0).

**Pull request merged**